tests/v2/validation/provisioning/rke2/etcd_backup_restore.go

```diff
@@ -31,14 +31,14 @@ const (
 	etcdnodeCount                = 3
 )
 
-func createSnapshot(client *rancher.Client, clustername string, generation int, namespace string) error {
+func CreateSnapshot(client *rancher.Client, clustername string, generation int, namespace string) error {
 	clusterObj, existingSteveAPIObj, err := getProvisioningClusterByName(client, clustername, namespace)
 	if err != nil {
 		return err
 	}
 
 	clusterObj.Spec.RKEConfig.ETCDSnapshotCreate = &rkev1.ETCDSnapshotCreate{
-		Generation: generation,
+		Generation: clusterObj.Spec.RKEConfig.ETCDSnapshotCreate.Generation + generation,
```

Comment:
lets make this + 1 and remove the integer generation from the function call as we need to update the snapshot by + 1 each time.
```suggestion
		Generation: clusterObj.Spec.RKEConfig.ETCDSnapshotCreate.Generation + 1,
```